buildenv/jenkins/JenkinsfileBase

```diff
@@ -575,7 +576,7 @@ def runTest( ) {
 					echo "env.DISPLAY is ${env.DISPLAY}"
 					makeTest("${RUNTEST_CMD}")
 				}
-				else if (env.SPEC.startsWith('linux')) {
+				else if (env.SPEC.startsWith('linux') && !LABEL.contains('ci.agent.dynamic')) {
```

Comment:
vm agent cannot set xvfb?

Reply:
This needs to be addressed. Otherwise, we will have issues running tests that need xvfb. If this is specific to a certain cloud, please update the check to be more specific.